js/sticky-on-scroll.js

```diff
@@ -102,12 +102,13 @@
 	  	if (typeof elScrollable === 'undefined') {
 	  		return;
 	  	}
-	    [].forEach.call( elScrollable, function( target ){
+
+    [].forEach.call( elScrollable, function( target ){
           target.style.height = 'auto';
 	        for(var i = 0; i < stickyElementDesktop.length ; i++ ) {
 	          heightSide = stickyElementDesktop[i].clientHeight;
 	        }
-	        heightElScrollable = target.clientHeight;
+	        heightElScrollable = document.querySelector('#conversation-detail-element-ul').clientHeight - posStickyTop;
```

Comment:
In my opinion, we should be use `getElementById`. It's faster than `querySelector` ;)

Reply:
👍